Row.c

```diff
@@ -495,7 +495,8 @@ void Row_printRate(RichString* str, double rate, bool coloring) {
 
 void Row_printLeftAlignedField(RichString* str, int attr, const char* content, unsigned int width) {
    int columns = width;
-   RichString_appendnWideColumns(str, attr, content, strlen(content), &columns);
+   size_t contentLen = strlen(content);
+   RichString_appendnWideColumns(str, attr, content, (int)MINIMUM(contentLen, 1000), &columns);
```

Comment:
I'm not sure I like this construct. If you want the cap the maximum length for a `strlen` output, then `strnlen` is there for use.

Reply:
Update: I've changed my mind on the use of `strnlen` here. While `strnlen` can guarantee it never reads past the specified `maxlen`, `strnlen` is (theoretically) less performant than `strlen` when the input string is well formed already.

That doesn't mean I deny the possibility of using `strnlen` in htop, but the length clamping, if needed, should be done as early as possible, e.g. in `Settings.c` when reading the strings from config and not long after the config file was read.

Just using `size_t` for `strlen` return values would be slightly cheaper than usijng strnlen`